build.gradle.kts

```diff
@@ -39,6 +39,9 @@ repositories {
   maven {
     url = uri("https://repo.maven.apache.org/maven2/")
   }
+  maven {
+    url = uri("https://aws.oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
Note that we will switch this in near future as this issue in progress:
* https://github.com/opensearch-project/opensearch-build/issues/5360